projects/kubernetes/release/1-23/GIT_TAG

```diff
@@ -0,0 +1 @@
+v0.12.0
```

Comment:
There is a newer version of this ((v0.13.0)[https://github.com/kubernetes/release/tree/v0.13.0]), but I thought we might want to just stay on where we were for 1.22. It wasn't super clear from upstream (https://github.com/kubernetes/kubernetes/blob/v1.23.6/build/common.sh#L94)

Reply:
I'll look at this one with you